### PR TITLE
update the doc for paddle.to_tensor

### PR DESCRIPTION
**Pull request by zhwesky2010**
### PR types

Bug fixes
### PR changes

APIs
### Describe

update the doc of 2.0 API for paddle.to_tensor, which is more easier to use.

### CODE REVIEW

File: python/paddle/tensor/logic.py [hunk 1]
```diff
@@ -75,9 +75,9 @@ def equal_all(x, y, name=None):
           import paddle
 
           paddle.disable_static()
-          x = paddle.to_variable(np.array([1, 2, 3]))
```
Comment (zhiqiu) [inline python/paddle/tensor/logic.py:78]: Better `x = paddle.to_tensor([1, 2, 3])`, same for others.

Comment (zhwesky2010) [reply python/paddle/tensor/logic.py:78]: Ok

**Review by jzhang533 — DISMISSED** [review]
lgtm

File: python/paddle/tensor/logic.py (hunk 1 above)
Comment (zhwesky2010) [reply python/paddle/tensor/logic.py:78]: All Done

**Review by zhiqiu — APPROVED** [review]

**Review by jzhang533 — APPROVED** [review]
lgtm

**Pull request merged**